.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,74 @@
+name: Evmone Coverage Report
```

Comment:
I have not reviewed this file yet, but I think we need to split this file into its own PR to have it properly reviewed, since it's completely orthogonal to test implementation.

Reply:
Yes, but on this pr, I can test directly that it works with the given test changes.

It's not ready yet.